src/rev_ai/customvocabulariesclient.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+"""Speech recognition tools for using Rev.ai"""
+
+import requests
+import json
+from requests.exceptions import HTTPError
+# from .models import Job, Account, Transcript, CaptionType
+from . import __version__
+
+try:
+    from urllib.parse import urljoin
+except:
+    from urlparse import urljoin
+
+
+class RevAiCustomVocabulariesClient:
+    """Client which implements Rev.ai CustomVocabulary API"""
+
+    # Default version of Rev.ai
+    version = 'v1'
+
+    # Default address of the API
+    base_url = 'https://api.rev.ai/speechtotext/{}/vocabularies'.format(
+        version)
+
+    def __init__(self, access_token):
+        """Constructor
+
+        :param access_token: access token which authorizes all requests and
+                             links them to your account. Generated on the
+                             settings page of your account dashboard
+                             on Rev.ai
+        """
+        if not access_token:
+            raise ValueError('access_token must be provided')
+
+        self.default_headers = {
+            'Authorization': 'Bearer {}'.format(access_token),
+            'User-Agent': 'RevAi-PythonSDK/{}'.format(__version__)
+        }
+
+    def submit_custom_vocabularies(
+            self,
+            custom_vocabularies,
```

Comment:
Another question: Should we define a Python object for custom_vocabularies or do we leave it open as generic object?

In my opinion, SDKs are developer guides and guides don't leave anything open for interpretation.

Reply:
But I hear from Kyle that it is Python standard to use raw objects instead of named object wrappers. You should talk to Kyle to make the final decision on this.

Reply:
I think in this case we should make this a list of "CustomVocabObjects". I guess to clarify a bit more, as far as I can tell, in python it's not normal to wrap `options` or compound objects in a single object to be used as an argument. Arguments are usually built in types or close to built in types (int, string, list). 

In this case `custom_vocabularies` is essentially a list of dictionaries, since all the dictionaries have the same key: `phrases` I think it makes sense to wrap that dictionary in an object

I don't know the formal reasoning for this but I would guess it's because python lets you use named arguments in functions so having a wrapper object around options doesn't really simplify anything like it does in node

Reply:
I think Kostas has arrive at a solution, @kbridbur if this is satisfactory will you resolve this comment?